src/platform/graphics/webgl/webgl-render-target.js

```diff
@@ -150,6 +150,12 @@ class WebglRenderTarget {
 
         } else {
 
+            Debug.call(() => {
```

Comment:
This will just warn though.  It won't resize the render targets back right?

Reply:
true.
this will cause webgl errors as well.
as discussed in the previous duplicate issue, the engine does not support this way of hiding the application, and it should not be used.

Reply:
it does work without the post though. If you do the same thing without the camera frame it will work correctly

Reply:
yes because normally we do not render to any render target
with camera frame or old post effects we render to render target, and currently we do not support when those are 0 size